Documentation/contributing/development/dev_setup.rst

```diff
@@ -609,6 +609,15 @@ This allows you to use a preconfigured development environment in the cloud or l
 The container is based on the official Cilium builder image and provides all the dependencies
 required to build Cilium.
 
+You can also install the required packages with ``contrib/scripts/devcontainer-setup.sh``.
```

Comment:
the required packages -> the common packages (e.g. kind, kubectl, and cilium-cli)

Reply:
https://github.com/cilium/cilium/pull/31534/commits/b6b694693a6d41ba6b2b5f2650407f2244ae4a01 fixed.